tests/jobs/test_backfill_job.py

```diff
@@ -1525,7 +1544,7 @@ def test_update_counters(self, dag_maker, session):
         # match what's in the in-memory ti_status.running map. This is the same
         # for skipped, failed and retry states.
         ti_status.running[ti.key] = ti  # Task is queued and marked as running
-        ti._try_number += 1  # Try number is increased during ti.run()
+        ti.try_number += 1  # Try number is increased during ti.run()
```

Comment:
This comment isn't right anymore, right?

Reply:
yep

```suggestion
        ti.try_number += 1
```